src/plugins/apis.js

```diff
@@ -28,7 +30,7 @@ const storeModule = {
       for (const api in state.urls) {
         const apiBaseURL = apiUrlFromRequestHeaders(api, req.headers);
 
-        if (apiBaseURL && this.$apis) {
+        if (apiBaseURL && this.$apis && this.$apis[api].$axios) {
```

Comment:
Let's use the optional chaining operator `apiBaseURL && this.$apis?.[api].$axios`

Reply:
Done.